jobs/kube-apiserver/templates/config/bpm.yml.erb

```diff
@@ -29,37 +29,43 @@ processes:
   - --cloud-provider=<%= cloud_provider.p('cloud-provider.type') %>
   - --cloud-config=/var/vcap/jobs/kube-apiserver/config/cloud-provider.ini
   <% end %>
+  - --enable-bootstrap-token-auth
```

Comment:
This feature appears to still be beta in 1.9: https://github.com/kubernetes/features/issues/43

Do we possibly want to wait on using this until it is GA in 1.10?